config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
```

Comment:
@MNisarAli Great work I love the way you have used resources to make the routes simple and easier to read and also added your users index route as your root route .